src/module.c

```diff
@@ -5654,6 +5654,8 @@ void RM_SetContextUser(RedisModuleCtx *ctx, const RedisModuleUser *user) {
  *     "3" -> REDISMODULE_ARGV_RESP_3
  *     "0" -> REDISMODULE_ARGV_RESP_AUTO
  *     "C" -> REDISMODULE_ARGV_RUN_AS_USER
+ *     "C" -> REDISMODULE_ARGV_CHECK_ACL
+ *     "O" -> REDISMODULE_ARGV_ALLOW_OOM_IN_SCRIPTS
```

Comment:
leftovers?

Reply:
yes.  but didn't have M so replaced it with M

Reply:
and a bad rebase that I didn't catch